### ISSUE TITLE
Notifications don't always respect dynamic text

### ISSUE THREAD

**Issue by koke** [wordpress-mobile/WordPress-iOS#16357]
It seems like the notifications list adapts to larger text sizes but the notification details don't, at leas for likes/comments:

![notifications-dynamic-text](https://user-images.githubusercontent.com/8739/115666732-3ad46100-a345-11eb-8504-54c9f9bade59.png)

### Steps to reproduce the behavior

1. [Switch the text size](https://support.apple.com/en-us/HT202828) to a larger font
2. Browse the notifications tab and notice that not everything adapts

**Comment by ScoutHarris** [comment]
Dupe of #9201. Also, Like notifications should be resolved with the Like Enhancements project (pbArwn-1CQ-p2, which is in progress).
